multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -24,20 +31,672 @@ use node_lib::database::db_initializer::{
+    let gas_price = 50_000_000_000_u64;
```

Comment:
Suggestion from Bert to Bert: Either change the name to `gas_price_in_wei` or provide it in `gwei` (since it's the standard), and later convert it into the desired unit.

By me 😉: At any path you decide, I'd still like the unit in the variable name.